galasa-parent/dev.galasa.framework.api.ras/src/test/java/dev/galasa/framework/api/ras/internal/TestRunArtifactsDownloadServlet.java

```diff
@@ -0,0 +1,233 @@
+/*
+ * Copyright contributors to the Galasa project 
+ */
+package dev.galasa.framework.api.ras.internal;
+
+import java.util.ArrayList;
+import java.util.List;
```

Comment:
Any idea what code coverage we get from these unit tests ?

Reply:
Code coverage is at 72% for the /ras/* apis